Content.Server/GameTicking/GameTicker.cs

```diff
@@ -140,6 +141,8 @@ public override void Initialize()
 
             SetStartPreset(_configurationManager.GetCVar(CCVars.GameLobbyDefaultPreset));
 
+            _roundInitializedTimeUtc = DateTime.UtcNow;
```

Comment:
When you've changed `_roundInitializedTimeUtc` to use `TimeSpan`, this should be set to `_gameTiming.RealTime`.